server/signature.go

```diff
@@ -28,6 +31,9 @@ func init() {
 
 func SetPrivateKey(s string) {
 	privateKeyFile = s
```

Comment:
The `privateKeyFile` package variable can be removed if we always call `SetPrivateKey` from `main.go`, either from `flagPrivateKey`, or from `PRIVATE_KEY` envvar. It's also more modular if `server` package doesn't depend on any envvar.

It also puzzled me that the `go test` in `server` package require `PRIVATE_KEY`. We can just save the test private key to a separate file and call `SetPrivateKey` in test setup.